doc/WebRTC.xml

```diff
@@ -672,6 +692,39 @@ Server -> Device: { "method": "trickle", "params": {"session": "s1", "candidate"
       <para>For two-way audio it's recommended to use echo cancellation to avoid feedback
         loops.</para>
     </section>
+    <section xml:id="section_data_channels">
+      <title>Data channels</title>
+        <section xml:id="section_data_channels_enable">
+          <title>Enabling data channels</title>
+            <para>If the device supports WebRTC data channels, it shall signal this in the SDP offer 
+              of the invite method. If the client supports data channels and intends to use them during 
+              the WebRTC session, it shall signal this in the SDP answer.</para>
+            <para>ONVIF-defined data channel subprotocol names shall have “vnd.onvif” prefix.  
+              Usage of this prefix is restricted to ONVIF-defined subprotocols only.</para>
+            <para>Before creating and using any data channel subprotocol, the device and client shall 
+              exchange a list of allowed data channel subprotocols in the invite method. Only 
+              subprotocols provided by both the device and client during the invite method shall be 
+              allowed to be enabled during the WebRTC session.</para>
+            <para>At any time after obtaining a WebRTC connection, a client may request enabling a 
+              pre-negotiated data channel subprotocol by creating a new data channel with the specified 
+              subprotocol identifier. This triggers a new data channel event on the device, which shall 
+              enable the requested data channel subprotocol only after ensuring it was negotiated during 
+              the SDP exchange. Events for non-pre-negotiated subprotocols shall be ignored.
+            </para>
+        </section>
+        <section xml:id="section_data_channels_metadata">
+          <title>Metadata over data channels</title>
+          <para><emphasis role="bold">Subprotocol identifier</emphasis>: vnd.onvif.metadata+gzip</para>
+            <para>The Metadata data channel subprotocol enables streaming XML Metadata over WebRTC data 
+              channels. Only GZIP compression is supported; uncompressed XML Metadata is not supported.</para>
+            <para>A device shall use the MetadataConfiguration from the media streaming profile selected 

```

Comment:
Suggest to rephrase to positive requirement statement:

An ONVIF compliant device shall support gzip compressed metadata as signaled via the subprotocol vnd.onvif.metadata+gzip.

This automatically implies that there is no guarantee for devices supporting uncompressed transmission.